src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -258,8 +264,10 @@
       <IDSymbol name="IDG_VENUS_CTX_REFERENCE" value="27" />
       <IDSymbol name="IDG_CTX_REFERENCE" value="0x102" />
     </GuidSymbol>
+    <!--guidReferenceContext is aka: ShellMainMenu_guid -->
```

Comment:
If guidReferenceContext isn't referenced elsewhere just call it guidSHLMainMenu which is what its called everywhere else, otherwise, you can fix this name and fix all references.

Reply:
@davkean thanks for the suggestion. I would prefer to do refactoring as separate work. I agree it needs to be done. I wrote up a task and will work on it this Sprint https://github.com/NuGet/Home/issues/9457